python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -779,8 +799,10 @@ def _evaluate_sensor(
 
         yield
     else:
-        yield from _handle_run_requests(
-            raw_run_requests=sensor_runtime_data.run_requests,
+        yield from _handle_run_requests_and_automation_condition_evaluations(
+            raw_run_requests=sensor_runtime_data.run_requests or [],
+            automation_condition_evaluations=sensor_runtime_data.automation_condition_evaluations
+            or [],
```

Comment:
i don't think you need the "or []" here right? since the object initializes them to [] if they are None?

Reply:
true true -- updated